portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -61,7 +61,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get BlogMessages from Datastore.
-    List<BlogMessage> blogMessages = LoadAllBlogsOrLast(true);
+    List<BlogMessage> blogMessages = LoadAllBlogsOrLast(/**all=*/ true);
```

Comment:
For nested comment, you would probably want to just use one star like /*all=*/

Reply:
Looks like the markup text is formatting Peizhao's comment maybe like:
/\*all=\*/